dashboard/app/controllers/reference_guides_controller.rb

```diff
@@ -14,6 +14,17 @@ def edit_all
   def show
   end
 
+  # GET /courses/:course_name/guides
+  def index
+    # redirect to the show page for the first guide within a category
+    course_version_id = find_matching_course_version(params[:course_course_name])&.id
+    first_category_key = ReferenceGuide.where(course_version_id: course_version_id, parent_reference_guide_key: nil).
+      order('position').first&.key
+    first_child_key = ReferenceGuide.where(course_version_id: course_version_id, parent_reference_guide_key: first_category_key).
+      order('position').first&.key
+    redirect_to course_reference_guide_path(params[:course_course_name], first_child_key)
```

Comment:
The test confirmed what I was wondering here. Why are we skipping over the parent guide to the child guide? The parent guide could have content in it.

Reply:
hmm, those pages do have a small amount of content, but right now as the nav bar is designed, there's no link to it on the page. I'm checking with tess to see if we want to move that somewhere else or change up the nav bar

Reply:
Interesting I did not realize that you could not visit those pages. That seems like a change from CB

Reply:
Okay the plan is to create new "Game Lab Overview" etc guides that will contain that content once the "create new guide" button is hooked up. So this logic should continue to skip over the top-level guides

Reply:
Sounds good!